docs/quick-start-guide.md

```diff
@@ -91,32 +90,31 @@ val carbon = SparkSession.builder().config(sc.getConf)
 ###### Creating a Table
 
 ```
-scala>carbon.sql("CREATE TABLE
-                    IF NOT EXISTS test_table(
-                    id string,
-                    name string,
-                    city string,
-                    age Int)
-                  STORED AS carbondata")
+carbon.sql("""CREATE TABLE IF NOT EXISTS test_table(
+                id string,
+                name string,
+                city string,
+                age Int)
+              STORED AS carbondata""")
 ```
 
 ###### Loading Data to a Table
 
 ```
-scala>carbon.sql("LOAD DATA INPATH '/path/to/sample.csv'
-                  INTO TABLE test_table")
+carbon.sql("LOAD DATA INPATH '/path/to/sample.csv' INTO TABLE test_table")
```

Comment:
Can you optimize the path? it's hdfs path when creating CarbonSession, but it's local path when loading data